src/lib.rs

```diff
@@ -679,6 +679,18 @@ macro_rules! try_slice {
 
 pub(crate) use try_slice;
 
+macro_rules! try_slice_mut {
+    ( $ptr:expr, $count:expr ) => {
+        if $ptr.is_null() {
+            return $crate::panic::NullParameterOrDefault::value();
+        } else {
+            unsafe { slice::from_raw_parts_mut($ptr, $count as usize) }
```

Comment:
Can we avoid this `as usize`, to prevent unsafety if someone calls it with (for example) `-1isize`? Let's assume $count is already the right type?

Reply:
I guess the same comment also applies to try_slice!

Reply:
Done. I also made the change to the original `try_slice!` macro I lifted this from.